lib/pf/pki_provider/inverse.pm

```diff
@@ -0,0 +1,141 @@
+package pf::pki_provider::inverse;
+
+=head1 NAME
+
+pf::pki_provider::inverse
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::pki_provider::inverse
+
+=cut
+
+use strict;
+use warnings;
+use Moo;
+use WWW::Curl::Easy;
+use URI::Escape::XS qw(uri_escape uri_unescape);
+
+use pf::log;
+
+=head2 uri
```

Comment:
Document the parameters below so we know what we're playing with if we develop in this module